### PR TITLE
fix line spacing

### PR DESCRIPTION
**Pull request by chashikajw**
### Purpose
As $subject, this will fix the line spacing

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

**Review by Rajith90 — APPROVED** [review]

**Comment by Rajith90** [comment]
@chashikajw please resolve merge conflicts

**Comment by chashikajw** [comment]
> @chashikajw please resolve merge conflicts

Resolved

**Comment by bhathiya** [comment]
@chashikajw why is this still open?

**Comment by chashikajw** [comment]
> @chashikajw why is this still open?

Will merge after today's release

File: test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/HttpClientRequest.java [hunk 1]
```diff
@@ -19,7 +19,13 @@
 package org.wso2am.micro.gw.tests.util;
```
Comment (praminda) [inline test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/HttpClientRequest.java:21]:
```suggestion
```

File: adapter/pkg/oasparser/utills/openApiUtills_test.go [hunk 2]
```diff
@@ -63,7 +61,6 @@ func TestFindSwaggerVersion(t *testing.T) {
 			message: "when openAPi version is not provided",
 		},
 	}
```
Comment (praminda) [inline adapter/pkg/oasparser/utills/openApiUtills_test.go:63]: I think its better to keep a newline before starting a conditional section. so shall we keep this
```suggestion
	}

```

**Review by praminda — APPROVED** [review]

**Pull request merged**